dev/devicelab/lib/tasks/integration_tests.dart

```diff
@@ -237,6 +241,10 @@ class IntegrationTest {
         ]);
       }
 
+      if (withTalkBack) {
+        await enableTalkBack();
```

Comment:
Do we know here what device the test is targeting? Can we assert if someone accidentally sets `withTalkBack` to true on a device that doesn't support it (e.g. iOS)?